### PR TITLE
Added exo2nek converter

### PR DESCRIPTION
**Pull request by stgeke**

### CODE REVIEW

File: tools/maketools [hunk 1]
```diff
@@ -8,8 +8,11 @@ bin_nek_tools="$HOME/bin"
 F77="gfortran"
 CC="gcc"
 
+# linking flags
+#USR_LFLAGS="-L$HOME/lib"
+
 # enable BIG MEMORY suuport
-BIGMEM="true"
+BIGMEM="false"
```
Comment (maxhutch) [inline tools/maketools]: Does this affect any of the other tools?

File: tools/exo2nek/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+   - For 2d meshes use z=0
+   - Periodic BC's are not supported
+   - Conjugate heat transfer (v and t-mesh) is not supported
+
+## BUILD INSTRUCTIONS
+1.  Download latest netcfd-c release from http://www.unidata.ucar.edu/downloads/netcdf/index.jsp
+2.  Modify the following defines in include/netcdf.h (recommended for use in exodus)
+    #define NC_MAX_DIMS     65536 
+    #define NC_MAX_VARS     524288
+    #define NC_MAX_VAR_DIMS 8
```
Comment (maxhutch) [inline tools/exo2nek/README.md]: You can wrap these in triple ticks to get block formatting, e.g.:

```
    #define NC_MAX_DIMS     65536 
    #define NC_MAX_VARS     524288
    #define NC_MAX_VAR_DIMS 8 
```

File: tools/maketools (hunk 1 above)
Comment (stgeke) [inline tools/maketools]: This was just for testing, forgot to switch it back.

**Comment by stgeke** [comment]
@maxhutch I followed your suggestions. Do you want to accept the PR?

**Comment by maxhutch** [comment]
It looks good to me, but I don't know anything about meshes or how to add a test.  Is there another dev that could have a look?  If not, then I'll merge it tomorrow.

**Comment by oanaoana** [comment]
Is this the converter from George! I thought we agreed in the meeting it would not be made public.

**Comment by maxhutch** [comment]
IIRC, Stefan wasn't at the meeting yesterday.

**Comment by stgeke** [comment]
Yes, that's the converter George developed. He told me he is happy to share the code with the community. It's still in an experimental stage and yes we it would be great to have a test case including an example. However this is not critical as PR only adds a stand-alone tool.

**Pull request merged**